src/document.jl

```diff
@@ -127,7 +129,7 @@ function get_offset2(doc::Document, line::Integer, character::Integer)
         pos = nextind(text, pos)
     end
 
-    return pos
+return pos
```

Comment:
These whitespace changes are a regression in DocumentFormat.jl, I think. Should figure out what's going on there...

Reply:
Argh, I didn't even notice these when I created the PR!